docs/release-notes.md

```diff
@@ -10,6 +10,77 @@ toc_max_heading_level: 2
 
 Welcome to Semgrep release notes. This document provides an overview of the changes, additions, and selected important fixes. Release notes published since April 2022 include Semgrep CLI, CI, and Semgrep App updates. Release notes published since April 2022 also include updates from more versions collected together.
 
+## July 2022
+
+### Semgrep CLI
+
+These release notes include upgrades for all versions ranging between 0.102.0 and 0.107.0.
+
+#### Additions
+
+- Semgrep CI:
+  - Fail-open support: Added `--suppress-errors` and `--no-suppress-errors` (the default is `--no-suppress-errors`).
+  - Support for podman environments!
```

Comment:
@bmahe is this correct wording here? Taken from the mention about the compare script: https://github.com/returntocorp/semgrep/blob/develop/CHANGELOG.md#added-2

Reply:
```suggestion
  - The `cli/scripts/compare.py` to compare rules for different versions of Semgrep is now supported on podman environments (compare-script-podman).
```

Reply:
Clarified with @adamkvitek on the call that podman support was added to the script.